test/export/test_export.py

```diff
@@ -112,6 +112,20 @@ def f(x, y):
         inp = ([torch.ones(1, 3)], torch.ones(1, 3))
         self._test_export_same_as_eager(f, inp)
 
+    def test_basic_non_strict(self):
```

Comment:
can we also add a test for fake tensor mode and fake tensor inputs?

Reply:
sure.